lib/workers/repository/onboarding/common.ts

```diff
@@ -7,6 +7,7 @@ export function getSemanticCommitPrTitle(config: RenovateConfig): string {
   return `${config.semanticCommitType ?? 'chore'}: ${config.onboardingPrTitle}`;
 }
 
+const configFileNames = getConfigFileNames();
```

Comment:
same as other places, user config isn't available at this stage

Reply:
`configFileNames` are present in the file config.